doc/tutorial/python/tutorial-install-python-bindings.dox

```diff
@@ -672,26 +672,50 @@ If you have an issue that looks like:
                   from /usr/include/c++/11/bits/specfun.h:45,
                   from /usr/include/c++/11/cmath:1935,
                   from /usr/include/c++/11/math.h:36,
-                  from /home/sfelton/miniconda3/envs/wrapper3.9/include/python3.9/pyport.h:205,
-                  from /home/sfelton/miniconda3/envs/wrapper3.9/include/python3.9/Python.h:50,
-                  from /home/sfelton/.local/include/pybind11/detail/common.h:266,
-                  from /home/sfelton/.local/include/pybind11/attr.h:13,
-                  from /home/sfelton/.local/include/pybind11/detail/class.h:12,
-                  from /home/sfelton/.local/include/pybind11/pybind11.h:13,
-                  from /home/sfelton/software/visp_build/modules/python/bindings/src/robot.cpp:3:
+                  from /home/user/miniconda3/envs/wrapper3.9/include/python3.9/pyport.h:205,
+                  from /home/user/miniconda3/envs/wrapper3.9/include/python3.9/Python.h:50,
+                  from /home/user/.local/include/pybind11/detail/common.h:266,
+                  from /home/user/.local/include/pybind11/attr.h:13,
+                  from /home/user/.local/include/pybind11/detail/class.h:12,
+                  from /home/user/.local/include/pybind11/pybind11.h:13,
+                  from /home/user/software/visp_build/modules/python/bindings/src/robot.cpp:3:
   /usr/include/c++/11/type_traits: **In instantiation of ‘struct std::is_move_constructible<vpImage<double> >’:**
   /usr/include/c++/11/type_traits:152:12:   required from ‘struct std::__and_<std::is_move_constructible<vpImage<double> >, std::is_move_assignable<vpImage<double> > >’
   /usr/include/c++/11/type_traits:157:12:   required from ‘struct std::__and_<std::__not_<std::__is_tuple_like<vpImage<double> > >, std::is_move_constructible<vpImage<double> >, std::is_move_assignable<vpImage<double> > >’
   /usr/include/c++/11/type_traits:2209:11:   required by substitution of ‘template<class ... _Cond> using _Require = std::__enable_if_t<std::__and_< <template-parameter-1-1> >::value> [with _Cond = {std::__not_<std::__is_tuple_like<vpImage<double> > >, std::is_move_constructible<vpImage<double> >, std::is_move_assignable<vpImage<double> >}]’
   /usr/include/c++/11/bits/move.h:196:5:   required by substitution of ‘template<class _Tp> std::_Require<std::__not_<std::__is_tuple_like<_Tp> >, std::is_move_constructible<_Tp>, std::is_move_assignable<_Tp> > std::swap(_Tp&, _Tp&) [with _Tp = vpImage<double>]’
-  /home/sfelton/software/visp-sfelton/modules/core/include/visp3/core/vpImage.h:341:15:   required from ‘class vpImage<double>’
-  /home/sfelton/software/visp-sfelton/modules/core/include/visp3/core/vpImage.h:369:17:   required from here
+  /home/user/software/visp-user/modules/core/include/visp3/core/vpImage.h:341:15:   required from ‘class vpImage<double>’
+  /home/user/software/visp-user/modules/core/include/visp3/core/vpImage.h:369:17:   required from here
```

Comment:
Same here, instead of `/home/user/software/visp-user/` use `/home/user/visp_ws/visp/`

Reply:
Done in 407c96aa0b683ac4183b234ca0e5d6000ef4d22c